### ISSUE TITLE
Canada wordmark is not read by Talkback

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#525]
Samsung Galaxy J3 Prime
Android version 7.0

On the splash screen, no text alternative is provided for the Canada wordmark image as Talkback doesn't say anything.

Expected result
The text alternative for the image should be available.

WCAG
https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html

Screenshots
![splash](https://user-images.githubusercontent.com/12256067/86924567-a3778780-c0fd-11ea-988c-7674970487f6.gif)

**Comment by nickmly** [comment]
@JuliannaR is this a duplicate of #457?

**Comment by JuliannaR** [comment]
Its the same issue, but its the watermark instead of the main splash image. We need to do some testing because the splash screen doesn't stay live long enough to listen to the alt text.

**Comment by nickmly** [comment]
According to this: https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/launch-screen

> Avoid including text on your launch screen. Because the content in a launch screen doesn't change, any displayed text won't be localized.

It seems like localized text on the splash screen is not possible on iOS, therefore we cannot do any alt text for it.

**Comment by JuliannaR** [comment]
Lets close this for now and revisit if we need to find a solution re: alt text on splash screen. #457 (comment)

https://github.com/cds-snc/covid-shield-mobile/issues/525#issuecomment-655715166
